lua/telescope/_extensions/lazy/actions.lua

```diff
@@ -96,6 +96,9 @@ function M.open_in_browser()
       return
     end
 
+    if telescope_lazy_config.opts.actions_opts.open_in_browser.auto_close then
```

Comment:
Ah, there might was missunderstanding here, I think it makes more sense if this is moved "below" the `if ret <= 0` check, as requested, but it got added above. 😁 (Make sure to separate by a newline)